LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Model/DataBroker.swift

```diff
@@ -84,14 +84,22 @@ extension MirrorSite {
     func scannedBroker(withStatus status: ScannedBroker.Status) -> ScannedBroker {
         ScannedBroker(name: name, url: url, status: status)
     }
+    
+    func shouldWeIncludeMirrorSite(for date: Date = Date()) -> Bool {
```

Comment:
I know it was a preexisting function, but what does this mean?

Reply:
It’s used when calculating if a mirror site should be included as a result in scan results. i.e - Profile is found on parent site A, but it also was found on mirror site B at the same time, so that is two results. I can’t fully speak to the reasoning, but it’s used in existing result calculations, so I think it should be use here also. I’ve added documentation to the method.